dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/FeatureUtility.java

```diff
@@ -121,19 +134,39 @@ private FeatureUtility(FeatureUtilityBuilder builder) throws IOException, Instal
         Set<String> envMapKeys = envMap.keySet();
         for (String key: envMapKeys) {
         	if (key.equals("FEATURE_REPO_PASSWORD")) {
-        		fine("FEATURE_REPO_PASSWORD: *********");
+		    fine("FEATURE_REPO_PASSWORD: *********");
         	} else if (key.equals("FEATURE_LOCAL_REPO") && envMap.get("FEATURE_LOCAL_REPO") != null) {
-        		fine(key +": " + envMap.get(key));
-        		File local_repo = new File((String) envMap.get("FEATURE_LOCAL_REPO"));
-        		this.fromDir = local_repo;
-        	}else {
-        		fine(key +": " + (envMap.get(key)));
+		    fine(key + ": " + envMap.get(key));
+		    File local_repo = new File((String) envMap.get("FEATURE_LOCAL_REPO"));
+		    this.fromDir = local_repo;
+        	} else {
+		    fine(key + ": " + (envMap.get(key)));
         	}
         }
 	map.put(InstallConstants.JSON_PROVIDED, false);
         overrideEnvMapWithProperties();
+	// check verify value after overriding
+	envMap = (Map<String, Object>) map.get(InstallConstants.ENVIRONMENT_VARIABLE_MAP);
+	if (envMap.get("FEATURE_VERIFY") != null) {
+	    try {
+		VerifyOption env = VerifyOption.valueOf(((String) envMap.get("FEATURE_VERIFY")).toLowerCase());
```

Comment:
validate the value in a separate method for easier maintenance. Also, let's run the check once instead of once for syntax and once for env / property files, etc.